src/platform/packages/private/kbn-esql-editor/src/types.ts

```diff
@@ -86,16 +86,6 @@ export interface ESQLEditorProps {
   esqlVariables?: ESQLControlVariable[];
 }
 
-export interface JoinIndicesAutocompleteResult {
```

Comment:
We were repeating this type across our packages / plugin so I moved it to eesql-types